node/node_explorer.go

```diff
@@ -176,3 +176,34 @@ func (node *Node) GetTransactionsHistory(address, txType, order string) ([]commo
 	}
 	return hashes, nil
 }
+
+// GetStakingTransactionsHistory returns list of staking transactions hashes of address.
+func (node *Node) GetStakingTransactionsHistory(address, txType, order string) ([]common.Hash, error) {
+	addressData := &explorer.Address{}
```

Comment:
If the code is for this function is exactly same as `GetTransactionsHistory`, can't you just call it?

Reply:
yes, no need for even further code duplication